lib/Analysis/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_subdirectory(RotationAnalysis)
```

Comment:
It seems like `MulDepthAnalysis` and `SecretnessAnalysis` don't yet have a CMakeLists.txt (they were recently added)
```suggestion
add_subdirectory(MulDepthAnalysis)
add_subdirectory(RotationAnalysis)
add_subdirectory(SecretnessAnalysis)
```

Reply:
ok added it